--- trunk/Cerebrum/modules/no/Constants.py.orig	2010-10-17 14:14:11.000000000 +0200
+++ trunk/Cerebrum/modules/no/Constants.py	2010-10-17 14:14:35.000000000 +0200
@@ -110,11 +110,11 @@
         affiliation_student, 'privatist', 'Student, privatist')
 
     ## Affiliations for employees
-    affiliation_ansatt = _PersonAffiliationCode('ANSATT', 'Ansatt')
-    affiliation_status_ansatt_vitenskapelig = _PersonAffStatusCode(
-        affiliation_ansatt, 'vitenskapelig', 'Ansatt, vitenskapelig')
-    affiliation_status_ansatt_tekadm = _PersonAffStatusCode(
-        affiliation_ansatt, 'tekadm', 'Ansatt, teknisk-administrativ')
+    #affiliation_ansatt = _PersonAffiliationCode('ANSATT', 'Ansatt')
+    #affiliation_status_ansatt_vitenskapelig = _PersonAffStatusCode(
+    #    affiliation_ansatt, 'vitenskapelig', 'Ansatt, vitenskapelig')
+    #affiliation_status_ansatt_tekadm = _PersonAffStatusCode(
+    #    affiliation_ansatt, 'tekadm', 'Ansatt, teknisk-administrativ')
 
     spread_ldap_group = _SpreadCode(
         'group@ldap', Constants.Constants.entity_group,
