--- cerebrum-ntnu-1.07+svn10206.orig/setup.py
+++ cerebrum-ntnu-1.07+svn10206/setup.py
@@ -161,6 +161,7 @@
             if os.path.isabs(cere_path):
                 cere_path = cere_path[1:]
             cere_path = os.path.join(self.root, cere_path)
+        cere_path = "debian/cerebrum-ntnu" + cere_path
         f_out = open(cere_path, "w")
         python_dir = sysconfig.get_python_lib(prefix=self.install_dir)
         for line in f_in.readlines():
