--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/htdocs/jscript/base.js
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/htdocs/jscript/base.js
@@ -166,7 +166,7 @@
 
     var addCloseButton = function (el) {
         var cb = document.createElement('img');
-        cb.src = '/yui/build/container/assets/close12_1.gif';
+        cb.src = '/yui/container/assets/close12_1.gif';
         YD.addClass(cb, 'upperRight');
         el.appendChild(cb);
         return cb;
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/htdocs/allocationperiod.py
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/htdocs/allocationperiod.py
@@ -36,8 +36,8 @@
     page.title = _("Search for allocation Period(s)")
     page.set_focus("allocationperiod/search")
     page.links = _get_links()
-    page.search_fields = [("name", _("Name"),
-                          ("allocationauthority", _("Allocation Authority"),
+    page.search_fields = [("name", _("Name")),
+                          ("allocationauthority", _("Allocation Authority")),
                          ]
     page.search_action = '/allocationperiod/search'
     page.form_values = remembered
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/htdocs/project.py
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/htdocs/project.py
@@ -20,7 +20,7 @@
 
 import cherrypy
 
-from account import _get_links()
+from account import _get_links
 from gettext import gettext as _
 from lib.Main import Main
 from lib.utils import commit, commit_url, queue_message, object_link
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/templates/OUEditTemplate.tmpl
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/templates/OUEditTemplate.tmpl
@@ -113,9 +113,7 @@
 ##             the <option>, while the last part is the 
 ##             displayed text for the option
 
-#def select_ou($perspective, $name, $label=None, $selected=None,
-               $disabled=None, 
-               $extras=())
+#def select_ou($perspective, $name, $label=None, $selected=None, $disabled=None, $extras=())
     
     #if $selected is not None and not isinstance($selected, StringTypes):
         #set $selected = $selected.get_id()
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/templates/UserTemplate.tmpl
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/templates/UserTemplate.tmpl
@@ -29,9 +29,11 @@
 ## This tells us not to load the worklist.js 
 #attr $wl = 0
 ## The following hack removes the bottom div which includes the $wl.
-#def page_bottom:
+#def page_bottom
+#end def
 ## The following hack removes the tabview
-#def tabview:
+#def tabview
+#end def
 ## Show nice businessCard-like info about person/account
 #def viewInfo($account)
     #set $info = []
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/templates/FramesTemplate.tmpl
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/templates/FramesTemplate.tmpl
@@ -68,22 +68,22 @@
         <meta http-equiv="Cache-Control" content="max-age=0" />
         <link rel="shortcut icon" type="image/gif" href="/img/litenmedtekst72.gif"/>
         <!-- Include yui styles -->
-        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/build/reset-fonts-grids/reset-fonts-grids.css"/>
-        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/build/container/assets/container.css"/>
-        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/build/logger/assets/logger.css"/>
-        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/build/datatable/assets/datatable.css"/>
-        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/build/tabview/assets/tabview.css"/>
-        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/build/tabview/assets/border_tabs.css"/> 
+        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/reset-fonts-grids/reset-fonts-grids.css"/>
+        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/container/assets/container.css"/>
+        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/logger/assets/logger.css"/>
+        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/datatable/assets/datatable.css"/>
+        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/tabview/assets/tabview.css"/>
+        <link rel="stylesheet" type="text/css" charset="$charset" href="/yui/tabview/assets/border_tabs.css"/> 
 
         <!-- Include yui components -->
-        <script type="text/javascript" charset="$charset" src="/yui/build/utilities/utilities.js"></script>
-        <script type="text/javascript" charset="$charset" src="/yui/build/logger/logger-min.js"></script>
-        <script type="text/javascript" charset="$charset" src="/yui/build/container/container-min.js"></script>
-        <script type="text/javascript" charset="$charset" src="/yui/build/element/element-beta-min.js"></script>
-        <script type="text/javascript" charset="$charset" src="/yui/build/datasource/datasource-beta-min.js"></script>
-        <script type="text/javascript" charset="$charset" src="/yui/build/datatable/datatable-beta-min.js"></script>
-        <script type="text/javascript" charset="$charset" src="/yui/build/autocomplete/autocomplete-min.js"></script>
-        <script type="text/javascript" charset="$charset" src="/yui/build/tabview/tabview-min.js"></script> 
+        <script type="text/javascript" charset="$charset" src="/yui/utilities/utilities.js"></script>
+        <script type="text/javascript" charset="$charset" src="/yui/logger/logger-min.js"></script>
+        <script type="text/javascript" charset="$charset" src="/yui/container/container-min.js"></script>
+        <script type="text/javascript" charset="$charset" src="/yui/element/element-beta-min.js"></script>
+        <script type="text/javascript" charset="$charset" src="/yui/datasource/datasource-beta-min.js"></script>
+        <script type="text/javascript" charset="$charset" src="/yui/datatable/datatable-beta-min.js"></script>
+        <script type="text/javascript" charset="$charset" src="/yui/autocomplete/autocomplete-min.js"></script>
+        <script type="text/javascript" charset="$charset" src="/yui/tabview/tabview-min.js"></script> 
 ##
 ## Maybe an *ugly* fix for selects that are too narrow in ie.
 ## Check FormWidgets for more...   tk.
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/templates/OUViewTemplate.tmpl
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/templates/OUViewTemplate.tmpl
@@ -73,29 +73,29 @@
             #try
                 #set $parent = $ou.get_parent($perspective)
             #except NotFoundError
-            #else
-                <h4>$_('Parent in perspective')
-			$html_quote($perspective.get_name())</h4>
-                #if $parent
-                <div>$object_link($parent)</div>
-                #else
-                <div>$_('(Root of perspective)')</div>
-                #end if
+                #continue
             #end try
+            <h4>$_('Parent in perspective')
+	    	$html_quote($perspective.get_name())</h4>
+            #if $parent
+            <div>$object_link($parent)</div>
+            #else
+            <div>$_('(Root of perspective)')</div>
+            #end if
 
             ## children
             #try
                 #set $children = $ou.get_children($perspective)
             #except NotFoundError
-            #else
-                #if $children
-                    <h4>$_('Children in perspective')
-			$html_quote($perspective.get_name())</h4>
-                    #for $child in $children
-                    <div>$object_link($child)</div>
-                    #end for
-                #end if     
+                #continue
             #end try
+            #if $children
+                <h4>$_('Children in perspective')
+                    $html_quote($perspective.get_name())</h4>
+                #for $child in $children
+                <div>$object_link($child)</div>
+                #end for
+            #end if     
         #end for
     </div>
 #end def
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/templates/ActivationTemplate.tmpl
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/templates/ActivationTemplate.tmpl
@@ -34,9 +34,11 @@
 ## This tells us not to load the worklist.js 
 #attr $wl = 0
 ## The following hack removes the bottom div which includes the $wl.
-#def page_bottom:
+#def page_bottom
+#end def
 ## The following hack removes the tabview
-#def tabview:
+#def tabview
+#end def
 #from lib.Languages import Languages
 
 #def header()
@@ -51,26 +53,26 @@
         <link rel="shortcut icon" type="image/gif" href="/img/litenmedtekst72.gif"/>
         <!-- Include yui styles -->
 
-        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/build/reset-fonts-grids/reset-fonts-grids.css"/>
-        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/build/container/assets/container.css"/>
-        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/build/logger/assets/logger.css"/>
-        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/build/datatable/assets/datatable.css"/>
-        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/build/tabview/assets/tabview.css"/>
-        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/build/tabview/assets/border_tabs.css"/> 
+        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/reset-fonts-grids/reset-fonts-grids.css"/>
+        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/container/assets/container.css"/>
+        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/logger/assets/logger.css"/>
+        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/datatable/assets/datatable.css"/>
+        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/tabview/assets/tabview.css"/>
+        <link rel="stylesheet" type="text/css" charset="UTF-8" href="/yui/tabview/assets/border_tabs.css"/> 
 
 
         <!-- Include yui components -->
 
-        <script type="text/javascript" charset="UTF-8" src="/yui/build/utilities/utilities.js"></script>
-        <script type="text/javascript" charset="UTF-8" src="/yui/build/logger/logger-min.js"></script>
+        <script type="text/javascript" charset="UTF-8" src="/yui/utilities/utilities.js"></script>
+        <script type="text/javascript" charset="UTF-8" src="/yui/logger/logger-min.js"></script>
 
-        <script type="text/javascript" charset="UTF-8" src="/yui/build/container/container-min.js"></script>
-        <script type="text/javascript" charset="UTF-8" src="/yui/build/element/element-beta-min.js"></script>
-        <script type="text/javascript" charset="UTF-8" src="/yui/build/datasource/datasource-beta-min.js"></script>
-        <script type="text/javascript" charset="UTF-8" src="/yui/build/datatable/datatable-beta-min.js"></script>
-        <script type="text/javascript" charset="UTF-8" src="/yui/build/autocomplete/autocomplete-min.js"></script>
+        <script type="text/javascript" charset="UTF-8" src="/yui/container/container-min.js"></script>
+        <script type="text/javascript" charset="UTF-8" src="/yui/element/element-beta-min.js"></script>
+        <script type="text/javascript" charset="UTF-8" src="/yui/datasource/datasource-beta-min.js"></script>
+        <script type="text/javascript" charset="UTF-8" src="/yui/datatable/datatable-beta-min.js"></script>
+        <script type="text/javascript" charset="UTF-8" src="/yui/autocomplete/autocomplete-min.js"></script>
         <script type="text/javascript" charset="UTF-8"
-        src="/yui/build/tabview/tabview-min.js"></script> 
+        src="/yui/tabview/tabview-min.js"></script> 
 
         <!-- Include our styles -->
 
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/templates/FormTemplate.tmpl
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/templates/FormTemplate.tmpl
@@ -84,7 +84,7 @@
                         <input class="$cls" type="$field['type']" value="$field['value']" name="$field['name']" id="$field['name']" />
                     #end if
                     #if $field.get('help')
-                        <img alt="?" class="tt" src='/yui/build/container/assets/hlp16_1.gif' /><span class="nojs">$field['help']</span>
+                        <img alt="?" class="tt" src='/yui/container/assets/hlp16_1.gif' /><span class="nojs">$field['help']</span>
                     #end if
                 </div>
             #end for
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/templates/ErrorTemplate.tmpl
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/templates/ErrorTemplate.tmpl
@@ -31,8 +31,8 @@
             <title>Cereweb error</title>
             <link rel="stylesheet" type="text/css" href="/css/style.css" />
             <!-- Include yui components -->
-            <script type="text/javascript" src="/yui/build/utilities/utilities.js"></script>
-            <script type="text/javascript" src="/yui/build/logger/logger-min.js"></script>
+            <script type="text/javascript" src="/yui/utilities/utilities.js"></script>
+            <script type="text/javascript" src="/yui/logger/logger-min.js"></script>
 
             <!-- Include our javascript application -->
             <script type="text/javascript" src="/jscript/error.js"></script>
--- cerebrum-ntnu-1.07+svn10371.orig/spine/cereweb/lib/CallWSIdm.py
+++ cerebrum-ntnu-1.07+svn10371/spine/cereweb/lib/CallWSIdm.py
@@ -30,7 +30,7 @@
 ssl = 'yes' 
 
 # Her lurer vi ZSI litt for å få bruke ZSI.client._AuthHeader
-kw = {'tracefile': tracefile,'ssl': ssl,'auth':(ZSI.AUTH.zsibasic,user,passwd)}
+kw = {'tracefile': tracefile,'ssl': ssl,'auth':(ZSI.auth.AUTH.zsibasic,user,passwd)}
 
 def main():
     global locator

