--- /home/andreas/src/cerebrum/setup.py	2004-11-03 12:24:22.000000000 +0100
+++ setup.py	2005-01-05 18:50:57.000000000 +0100
@@ -162,6 +162,7 @@
 
     def run (self):
         self.mkpath(self.install_dir)
+        fp = open('/tmp/filelist', 'a')
         for f in self.data_files:
             # it's a tuple with dict to install to and a list of files
             tdict = f[0]
@@ -172,14 +173,14 @@
                 dir = change_root(self.root, dir)
             self.mkpath(dir)
             os.chmod(dir, tdict['mode'])
-            if(os.geteuid() == 0):
-                try:
-                    uinfo = pwd.getpwnam(tdict['owner'])
-                except KeyError:
-                    print "Error: Unkown user %s" % tdict['owner']
-                    sys.exit(1)
-                uid, gid = uinfo[2], uinfo[3]
-                os.chown(dir, uid, gid)
+#            if(os.geteuid() == 0):
+#                try:
+#                    uinfo = pwd.getpwnam(tdict['owner'])
+#                except KeyError:
+#                    print "Error: Unkown user %s" % tdict['owner']
+#                    sys.exit(1)
+#                uid, gid = uinfo[2], uinfo[3]
+#                os.chown(dir, uid, gid)
             if f[1] == []:
                 # If there are no files listed, the user must be
                 # trying to create an empty directory, so add the
@@ -192,8 +193,9 @@
                     (out, _) = self.copy_file(data, dir)
                     self.outfiles.append(out)
                     os.chmod(out, mode)
-                    if(os.geteuid() == 0):
-                        os.chown(out, uid, gid)
+                    fp.write('%s\n' % out)
+#                    if(os.geteuid() == 0):
+#                        os.chown(out, uid, gid)
 
 class test(Command):
     user_options = [('check', None, 'Run check'),
@@ -248,6 +250,7 @@
                    'Cerebrum/modules/no/uio/printer_quota',
                    'Cerebrum/modules/no/uio/AutoStud',
                    'Cerebrum/modules/no/feidegvs',
+                   'Cerebrum/modules/no/debian-edu',
                    'Cerebrum/modules/no/hia',
                    'Cerebrum/modules/templates',
                    'Cerebrum/client',
@@ -320,7 +323,8 @@
                        [('client/passweb.py', 0755),
                         ('client/passweb_form.html', 0644),
                         ('client/passweb_receipt.html', 0644),
-                        ('java/jbofh/dist/lib/JBofh.jar', 0644)]),
+#                        ('java/jbofh/dist/lib/JBofh.jar', 0644)
+                        ]),
                      ({'path': sysconfdir,
                        'owner': cerebrum_user,
                        'mode': 0755},
