Index: Cerebrum/default_config.py
===================================================================
RCS file: /cvsroot/cerebrum/cerebrum/Cerebrum/default_config.py,v
retrieving revision 1.70
diff -u -I^#.*python2\.[23]$ -r1.70 default_config.py
--- Cerebrum/default_config.py	6 Jun 2004 23:28:31 -0000	1.70
+++ Cerebrum/default_config.py	2 Aug 2004 11:06:52 -0000
@@ -167,14 +167,14 @@
 NOTES_SOURCE_SEARCH_ORDER = ('system_ureg','system_lt','system_fs')
 
 # You should set this variable to the location of your logging ini file
-LOGGING_CONFIGFILE = None
+LOGGING_CONFIGFILE = '/etc/cerebrum/logging.ini'
 
 QUARANTINE_RULES = {}
 # QUARANTINE_RULES = {
 #   'system': {'lock': 1, 'shell': '/local/etc/shells/nologin.system'}
 # }
 
-CEREBRUM_DDL_DIR="../share/doc/cerebrum/design"
+CEREBRUM_DDL_DIR="../share/cerebrum/design"
 BOFHD_SUPERUSER_GROUP=INITIAL_GROUPNAME
 BOFHD_STUDADM_GROUP=BOFHD_SUPERUSER_GROUP
 # Should contain mapping lang: [('template-prefix', 'tpl-type)...]

