--- /dev/null	2004-10-01 16:38:37.000000000 +0200
+++ doc/Makefile	2004-11-05 00:06:44.000000000 +0100
@@ -0,0 +1,40 @@
+# documentation build options
+
+CEREDOC = $(PWD) 
+CERESRC = $(PWD)/..
+SP_ENCODING = XML
+SP_CHARSET_FIXED = YES
+
+# generic settings
+
+DESTDIR     =
+prefix      = /usr
+
+docdir      = $(prefix)/share/doc/cerebrum-doc
+htmldir     = $(docdir)/html
+mandir      = $(prefix)/share/man
+man1dir     = $(mandir)/man1
+
+# there are no manpages yet, but they will be called
+MAN         = makedb.1 jbofh.1 bofhd.1
+
+INSTALL_DATA = install --mode=644
+
+
+
+all:
+
+install: 
+	install -d 	$(DESTDIR)$(docdir)  \
+			$(DESTDIR)$(htmldir) \
+			$(DESTDIR)$(man1dir)
+	$(INSTALL_DATA)	 
+	touch stamp-install
+clean:
+	rm -f blub
+	rm -f stamp-build stamp-html stamp-pdf
+
+build: 
+	CEREDOC=$(CEREDOC) CERESRC=$(CERESRC) SP_ENCODING=$(SP_ENCODING) SP_CHARSET_FIXED=$(SP_CHARSET_FIXED) python $(PWD)/scripts/build.py --build-contents contents.rst
+	CEREDOC=$(CEREDOC) CERESRC=$(CERESRC) SP_ENCODING=$(SP_ENCODING) SP_CHARSET_FIXED=$(SP_CHARSET_FIXED) python $(PWD)/scripts/build.py --xml book.xml
+	(cd build; db2pdf book.xml)
