Index: server/bofhd.py
===================================================================
RCS file: /cvsroot/cerebrum/cerebrum/server/bofhd.py,v
retrieving revision 1.82
diff -u -I^#.*python2\.[23]$ -r1.82 bofhd.py
--- server/bofhd.py	29 Jun 2004 15:04:39 -0000	1.82
+++ server/bofhd.py	2 Aug 2004 11:07:07 -0000
@@ -598,7 +598,7 @@
             if not line:
                 break
             line = line.strip()
-            if line[0] == '#' or not line:
+            if not line or line[0] == '#':
                 continue
             # Import module and create an instance of it; update
             # mapping from command name to a class instance with a

